_includes/releases/v22.1/v22.1.2.md

```diff
@@ -0,0 +1,61 @@
+## v22.1.2
+
+Release Date: June 22, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-2-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- CSV is now a supported format for changefeeds. This only works with `initial_scan='only'` and does not work with diff/resolved options. [#82355][#82355]
+
+<h3 id="v22-1-2-sql-language-changes">SQL language changes</h3>
+
+- The `bulkio.ingest.sender_concurrency_limit` [cluster setting](../v22.1/cluster-settings.html) can be used to adjust the concurrency at which any one SQL node, across all operations that it is running (e.g., [`RESTORE`s](../v22.1/restore.html), [`IMPORT`s](v22.1/import.html), and [schema changes](../v22.1/online-schema-changes.html)), will send bulk ingest requests to the KV storage layer. [#81789][#81789]
+- The `sql.ttl.range_batch_size` [cluster setting](../v22.1/cluster-settings.html) is deprecated. [#82711][#82711]
+- Row-level TTL now works for multi-tenant clusters. [#82711][#82711]
```

Comment:
I thought we weren't supposed to refer to `multi-tenant` in release notes.

Reply:
You're right; I completely forgot. Removing.